packages/std/src/lib.rs

```diff
@@ -90,7 +91,10 @@ mod imports;
 mod memory; // Used by exports and imports only. This assumes pointers are 32 bit long, which makes it untestable on dev machines.
 
 #[cfg(target_arch = "wasm32")]
-pub use crate::exports::{do_execute, do_instantiate, do_migrate, do_query, do_reply, do_sudo};
+pub use crate::exports::{
```

Comment:
should these callback entry points be ibc feature gated?

Reply:
I was wondering about that myself. The other ibc entrypoints are gated by the `stargate` feature, but I don't know if that makes sense here. Not all chains that support the stargate capability have callbacks support (and the stargate feature is a little weird anyways).
We could add a new capability for it, like `ibc-callbacks`. Then, contract devs get an error, unless they enable it and the chain needs to provide the capability for the contract to be able to be uploaded. That would prevent uploading contracts that expect callbacks to be uploaded on chains without it.
However, I really don't know if that makes sense. Callbacks support is not really a binary thing. A chain might support it for some modules and not for others. Opinions? @webmaster128 @srdtrk